index.js

```diff
@@ -58,15 +60,30 @@ class K8sVMExecutor extends Executor {
     /**
      * Starts a k8s build
      * @method start
-     * @param  {Object}   config            A configuration object
-     * @param  {Integer}  config.buildId    ID for the build
-     * @param  {String}   config.container  Container for the build to run in
-     * @param  {String}   config.token      JWT for the Build
+     * @param  {Object}   config                A configuration object
+     * @param  {Object}   config.annotations    Set of key value pairs
```

Comment:
Already passed in from models: https://github.com/screwdriver-cd/models/blob/master/lib/build.js#L158